js_modules/dagit/packages/core/src/schedules/ScheduleRoot.tsx

```
@@ -123,10 +124,16 @@ const SchedulePreviousRuns: React.FC<SchedulePreviousRunsProps> = (props) => {
     pollInterval: 15 * 1000,
   });
 
+  if (!data || data.pipelineRunsOrError.__typename !== 'Runs') {
+    return null;
+  }
```

are we swallowing here if there's an error? should we display it somewhere?